rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "domain": 1,
+        "gasOracle": {
+            "type": "remoteGasData",
+            "tokenExchangeRate": "15000000000000000000",
```

Comment:
isn't ETH the native token on eclipse as well? Not sure what this represents

Reply:
this is 1.5, which is the premium we charge on EVM